ceph_iscsi_config/client.py

```diff
@@ -234,6 +234,30 @@ def define_client(self):
                              "successfully".format(self.iqn))
             self.change_count += 1
 
+    @staticmethod
+    def get_client_info(target_iqn, client_iqn):
+        result = {
+            "alias": '',
+            "state": '',
+            "ip_address": []
+        }
+        iscsi_fabric = ISCSIFabricModule()
+        target = Target(iscsi_fabric, target_iqn, 'lookup')
+        for tpg in target.tpgs:
+            if tpg.enable:
+                for client in tpg.node_acls:
+                    if client.session and client.session['parent_nodeacl'].node_wwn == client_iqn:
+                        result['alias'] = client.session.get('alias')
+                        state = client.session.get('state').upper()
+                        result['state'] = state
+                        ips = set()
+                        if state == 'LOGGED_IN':
+                            for conn in client.session.get('connections'):
+                                ips.add(conn.get('address'))
+                            result['ip_address'] = list(ips)
+                            break
```

Comment:
Was this break meant to break us out of the "for client in tpg.node_acls:" loop? Why is it only done if the state is logged in?